code/game/objects/items/flag.dm

```diff
@@ -219,3 +219,47 @@
 	name = "Nar'Sie Cultist flag"
 	desc = "A flag proudly boasting the logo of the cultists, sworn enemies of NT."
 	icon_state = "cultflag"
+	
+//Chameleon
+
+/obj/item/flag/chameleon
+	name = "Chameleon flag"
+	desc = "A poor recreation of the official NT flag. It seems to shimmer a little."
+	icon_state = "ntflag"
+	origin_tech = "materials=3;magnets=4;syndicate=4"
+	var/time = 0
+	var/used = 0
+	
+/obj/item/flag/chameleon/New()
+	time = rand(0,30)
+	
+/obj/item/flag/chameleon/attack_self(mob/user as mob)
+	if(used)
+		return
```

Comment:
Makes people think about their choice of flag and cuts down a bit from people just spamming use on flags for meta.